File: inverse.cpp
// (a * x) mod m = 1
#include <bits/stdc++.h>
using namespace std;

int modInverse(int a, int m) {
	for(int x = 1; x < m; x++) {
		if(((a % m) * (x % m)) % m == 1) {
			return x;
		}
	}
}

int main() {
	int a, m;
	cin >> a >> m;
	cout << "x la " << modInverse(a, m);
}

File: inverse.py
# (a * x) mod m = 1
def modInverse(a, m):
    for x in range(1, m):
        if ((a % m) * (x % m)) % m == 1:
            return x
        
a = int(input("Nhap a: "))
m = int(input("Nhap m: "))
print("x = ", modInverse(a, m))